content/en/docs/how-to/plugin-management.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Install and manage notation plugins
+description: Install and manage notation plugins
+weight: 9
+---
+
+Plugins for `notation` provide additional functionality and integration. For example, the [notation-azure-kv](https://github.com/Azure/notation-azure-kv) plugin allows you to sign a Notation-generated payload with a private key in [Azure Key Vault](https://learn.microsoft.com/en-us/azure/key-vault/general/overview?wt.mc_id=azurelearn_inproduct_oss_notaryproject).
+
+## Install a plugin
+
+To install a plugin, download the plugin binary, create a `{plugin-name}` directory in `{NOTATION_LIBEXEC}/plugins/`, and add the plugin binary to that directory.
+
+{{% alert title="Warning" color="warning" %}}
+Before creating the `{plugin-name}` directory, confirm you are using a name that follows the [naming structure](#plugin-naming-structure) for plugins. Plugins that do not follow the naming structure will not be recognized by `notation`.
+{{% /alert %}}
+
+The following example downloads and installs version 0.6.0 of [notation-azure-kv](https://github.com/Azure/notation-azure-kv) plugin for on macOS with Apple Silicon using the default location for the Notation installation.
```

Comment:
Please add the below line to highlight that customers can get plugins bundled with  Notation client from their plugin vendors.

 You can also refer to the installation instructions from your plugin vendor, who may bundle the plugin and Notation binary together, example [AWS Signer](https://docs.aws.amazon.com/signer/latest/developerguide/image-signing-prerequisites.html)

Reply:
fixed